frontend/src/views/camp/Invitation.vue

```diff
@@ -1,66 +1,94 @@
 <template>
-  <auth-container>
-    <div v-if="!ready" class="text-center">
-      <v-progress-circular size="64" indeterminate color="primary" />
-    </div>
-    <div v-else-if="invitationFound === true">
-      <h1 class="display-1">
-        {{ $tc('views.camp.invitation.title', 0, { campName: invitation().campTitle }) }}
+  <div class="bg">
+    <div class="hill">
+      <v-icon :size="$vuetify.breakpoint.smAndUp ? 300 : 200" class="tent"
+        >$vuetify.icons.tentDay
+      </v-icon>
+      <img src="../../assets/invitation/tree-left.svg" alt="" class="tree l h-full" />
+      <img src="../../assets/invitation/tree-right.svg" alt="" class="tree r h-full" />
+      <div class="localnav justify-space-between d-flex col">
+        <ButtonBack v-if="!$vuetify.breakpoint.mdAndUp" text dark visible-label />
```

Comment:
Back button and User Avatar can get behind the ropes. Should rather be in front, right?
<img width="526" alt="Screenshot 2023-06-24 at 16 22 57" src="https://github.com/ecamp/ecamp3/assets/449555/4c570207-f4f7-4286-ad83-f53d6ce6eb9e">